packages/unenv-preset/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { cloudflare } from "./preset";
+export { env, nodeless } from "unenv";
```

Comment:
Is is likely that this will change before the final unenv v2 release to simpler `createEnv({ presets: [], ...options })` and we drop/deprecate older exports.

Totally up to you (as long as we have unenv as relaxed `^` peer dependency it is fine) but I would very much root for not doing reexport and compose where preset is "consumed" (it can be `wrangler`, esm.sh, nitro or other places using unenv and will be always forward compatible)

Reply:
I added your 2 comments to the tracking issue #7366

The idea is to get this in to unblock the next steps and then we can tweak.

Reply:
The issue is that, if we start by having peer dependency of unenv, even if unused, it keeps nitro blocked as it forces a pinned version of unenv. 

While it is actually possible that we keep unenv as a `wrangler` dependency so that all consumers are free to choose their versioning policy for `unenv` (the composer) + their preset compatibility.

Reply:
Having peer dep as a range in the future is workable. But we hack a dead-lock situation here.

- unenv v2 won't be released unless we can test it fully against nitro
- unenv-preset won't relax peer dep of unenv until unenv v2 is released
- nitro (nightly) cannot test unenv-preset without setting the pinned dep

I might be missing some of the context in your integration plan and steps but it looks to me, `0.0.1` can be safely released without reexports and peer dependency and tested/integrated against wrangler by composing in wrangler layer and this way starting `0.0.1` we can also start testing it against nitro to finalize both sides.